python/plugins/processing/core/Processing.py

```diff
@@ -58,6 +50,14 @@
 from processing.algs.taudem.TauDEMAlgorithmProvider import TauDEMAlgorithmProvider
 from processing.tools import dataobjects
 
+__author__ = 'Victor Olaya'
```

Comment:
$ pep8 python/plugins/processing/core/Processing.py
python/plugins/processing/core/Processing.py:28:1: E402 module level import not at top of file